src/workspace/workspace.service.ts

```diff
@@ -97,9 +96,10 @@ export class WorkspaceService {
   async findAllByUserId(pageOptionsDto: PageOptionsDto, userId: number): Promise<PageDto<DbWorkspace>> {
     const queryBuilder = this.baseQueryBuilder();
 
-    queryBuilder.innerJoinAndSelect("workspaces.members", "workspace_members", "workspace_members.user_id = :userId", {
-      userId,
-    });
+    queryBuilder.innerJoinAndSelect("workspaces.members", "workspace_members").where(`
+      "workspaces"."id" IN (
+        SELECT "workspace_id" FROM "workspace_members" WHERE "user_id" = '${userId}'
+      )`);
```

Comment:
Can we still use the named parameter to avoid using it directly in the SQL?
```suggestion
    queryBuilder.innerJoinAndSelect("workspaces.members", "workspace_members").where(`
      "workspaces"."id" IN (
        SELECT "workspace_id" FROM "workspace_members" WHERE "user_id" = :userId
      )`, { userId });
```